raster-foundry/raster-foundry
# Add annotate to action_type and update related project routes

## Overview

This PR adds `ANNOTATE` to `action_type` and updates related project routes for annotation create, update, and delete.

### Checklist

- ~Styleguide updated, if necessary~
- ~[Swagger specification](https://github.com/raster-foundry/raster-foundry-api-spec) updated~
- ~Any content changes are properly templated using `BUILDCONFIG.APP_NAME`~


## Testing Instructions

 * `mg update`
 * `mg apply`
 * `./scripts/psql`
 * `SELECT enum_range(NULL::action_type);` -- `ANNOTATE` should be there

Closes #3312
